### PR TITLE
TCE eligibility

### LINKED ISSUE #4823: TCE eligibility
https://www.irs.gov/individuals/free-tax-return-preparation-for-qualifying-taxpayers

In addition to VITA, the TCE program offers free tax help, particularly for those who are 60 years of age and older, specializing in questions about pensions and retirement-related issues unique to seniors.

### PR DESCRIPTION
**Pull request by shuyu7zhan**
Fixes #4823

### CODE REVIEW

File: policyengine_us/parameters/gov/irs/tce/eligibility/age_limit.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+description: The IRS extends the Tax Counseling for the Elderly (TCE) program to senior filers with age at or above this value in addition to VITA.
+values:
+  2021-01-01: 60
+metadata:
+  unit: 1/
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/tce/eligibility/age_limit.yaml:5]:
```suggestion
  unit: year
```

File: policyengine_us/parameters/gov/irs/tce/eligibility/age_limit.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  period: year
+  label: TCE Program Age Limit
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/irs/tce/eligibility/age_limit.yaml:7]:
```suggestion
  label: TCE program age limit
```

File: policyengine_us/variables/gov/irs/tce/tce_eligible.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+class tce_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "Eligible for the TCE program"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.irs.tce.eligibility
+        is_eligible_age = person("age", period) >= p.age_limit
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/tce/tce_eligible.py:12]:
```suggestion
        age = person("age", period
        return age >= p.age_limit
```

File: policyengine_us/variables/gov/irs/tce/tce_eligible.py [hunk 4]
```diff
@@ -0,0 +1,13 @@
+from policyengine_us.model_api import *
+
+
+class tce_eligible(Variable):
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/tce/tce_eligible.py:4]:
```suggestion
class is_tce_eligible(Variable):
```

File: policyengine_us/variables/gov/irs/tce/tce_eligible.py [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+    value_type = bool
+    entity = Person
+    label = "Eligible for the TCE program"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/irs/tce/tce_eligible.py:7]:
```suggestion
    label = "Eligible Person for the TCE program"
```

File: changelog_entry.yaml [hunk 6]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - IRS TCE Program Eligibility.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - IRS TCE Program eligibility.
```

**Review by PavelMakarchuk — CHANGES_REQUESTED** [review]

**Comment by vrathi101** [comment]
https://github.com/PolicyEngine/policyengine-us/issues/4751

**Comment by PavelMakarchuk** [comment]
Duplicate of #4751

**Pull request closed without merging**